### PR TITLE
add time periods of 2019-20 study year :thisisfine:

### PR DESCRIPTION
**Pull request by Gurrit**

### CODE REVIEW

**Review by Hoidi — APPROVED** [review]

**Comment by Portals** [comment]
Why not just add all the years now? :100:

**Comment by Gurrit** [comment]
@Portals it should be done properly instead, force a patet to do it during code and "whine"

**Review by molleer — APPROVED** [review]

File: app/helpers/stats_helper.rb [hunk 1]
```diff
@@ -36,11 +37,18 @@ module StatsHelper
...
+        [Time.new(2020,1,21), Time.new(2020,3,24)], #LP3
+        [Time.new(2020,3,25), Time.new(2020,6,8)], #LP4
+
     ]
 
     def get_current_study_year_index()
+      print "go here"
         @@study_years.each_with_index do |study_year, index|
             if study_year[0].beginning_of_day <= Time.now && Time.now <= study_year[1].end_of_day
+              puts(index)
```
Comment (Kalior) [inline app/helpers/stats_helper.rb:51]: ??????

File: app/helpers/stats_helper.rb [hunk 2]
```diff
@@ -36,11 +37,18 @@ module StatsHelper
...
 
+        [Time.new(2019,9,3), Time.new(2019,11,4)], #LP1
+        [Time.new(2019,11,5), Time.new(2020,1,20)], #LP2
+        [Time.new(2020,1,21), Time.new(2020,3,24)], #LP3
+        [Time.new(2020,3,25), Time.new(2020,6,8)], #LP4
+
     ]
 
     def get_current_study_year_index()
+      print "go here"
```
Comment (Kalior) [inline app/helpers/stats_helper.rb:48]: ????????

**Review by Kalior — COMMENTED** [review]
??????

**Pull request merged**